src/coreclr/System.Private.CoreLib/src/System/Runtime/RuntimeImports.CoreCLR.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace System.Runtime
+{
+    internal static class RuntimeImports
```

Comment:
These should be methods on GCFrameRegistration. I do not think we want to introduce RuntimeImports indirection in regular CoreCLR. It is against the established naming pattern for FCalls (I hope we will swtich native AOT to the same naming pattern at some point too).

Reply:
Done. This was an attempt to have similar code across all 3 runtimes - now just Core and Mono have the same location for these 2 methods.